README.md

```diff
@@ -106,6 +106,48 @@ cd Scripts
 ```
 
+
+### Mock assignment run in Helmet UI
+
+Refer to this manual: [https://hsldevcom.github.io/helmet-docs/kaytto-ohje.html#helmet-asennus](https://hsldevcom.github.io/helmet-docs/kaytto-ohje.html#helmet-asennus).
+
+#### Change the config settings
+1. Download latest version of helmet-model-system in `Projektin asetukset` (edit: or set this manually to `helmet-model-system\Scripts` folder which you probably downloaded when installing Python environment!).
+2. Find out where that version is saved by hovering over the text box (for example: `C:\Users\xxx\helmet-model-system-v4.1.3-24-04-1684924655775`)
+3. Open `dev-config.json` inside that previous folder and modify OPTIONAL_FLAGS argument:
+    -     `"OPTIONAL_FLAGS": ["DO_NOT_USE_EMME"]`
+	
+#### Set Helmet UI settings
+1. Click "Projektin asetukset" button
+    - Python from previous virtual environment, for example `C:\Users\xxx\.virtualenvs\Scripts-AJXyDTuP\Scripts\python.exe`
+    - helmet-model-system was set previously
+    - Projektin kansiopolku: your choice
+    - Lähtödata: `helmet-model-system\tests\test_data\Base_input_data`
+    - Tulokset: your choice (copy Matrices folder from to the location) or `helmet-model-system\tests\test_data\Results`
+2. "Uusi Helmet-skenaario" button
+    - Skenaarion nimi: your choice
+    - Emme-projekti: create an empty (dummy) text file and save it with file extension `.emp`
+    - Liikenneverkon sisältävä skenaario: default
+    - Syöttötiedot: `helmet-model-system\tests\test_data\Scenario_input_data\2030_test`
+    - Iteraatioiden enimmäismäärä: default (do not cross "loppusijoittelu")
+    - The rest can have default values (edited) 
+	
+#### Run your first demo model run
+1. In case your scenario name is not `test`, copy the `helmet-model-system\tests\test_data\Results\test\Matrices` folder to `[your-result-folder]\[your-scenario-name]\Matrices`.
+2. Select scenario with checkbox and click `Käynnistä (1) skenaariota`. When running the first model run again and again, just do #2. When you create more scenarios, always copy matrices (#1) first to a new location. (This is a demo model thing that needs to be done because we are independent of Emme.)
+	
+#### Running mock assignment in command line
+Alternatively, you can run the mock assignment with the help of command line
+1. Make sure your virtual environment is activated (As described in Environment and Dependencies)
+2. Adjust the following variables in `dev-config.json` (while replacing the YOUR_PATH variable with absolute path to the helmet-model-system folder)
+    - `"SCENARIO_NAME": "test"`,
+    - `"RESULTS_PATH": "YOUR_PATH\\helmet-model-system\\Scripts\\tests\\test_data\\Results"`,
+    - `"BASELINE_DATA_PATH": "YOUR_PATH\\helmet-model-system\\Scripts\\tests\\test_data\\Base_input_data"`,
+    - `"FORECAST_DATA_PATH": "YOUR_PATH\\helmet-model-system\\Scripts\\tests\\test_data\\Scenario_input_data\\2030_test"`,
+    - `"OPTIONAL_FLAGS": ["DO_NOT_USE_EMME"]`
```

Comment:
Just found this one, @hsl-petrhaj. We have previously, according to the remarks of @johpiip, instructed that `"OPTIONAL_FLAGS"` should **not** be used when running `helmet` from command line. And that is still what we state here: https://github.com/HSLdevcom/helmet-model-system/tree/olusanya/Scripts#optional_flags

The idea is that run-time flag parameter should always change the state of the flag from "off" to "on". If the flag is already set in `dev-config.json`, this is invalidated.

Reply:
Solved in readme_update pull request #545